### PR TITLE
Add new bidders from 1.5.0 thru 1.18.0

### PR DESCRIPTION
**Pull request by jsnellbaker**
Adding new bidders between Prebid.js releases 1.5.0 - 1.18.0

### CODE REVIEW

**Comment by mkendall07** [comment]
@matthewlane 
Can you work with @jsnellbaker for publish?

File: bidderPatterns.js [hunk 1]
```diff
@@ -412,5 +413,118 @@ var bidderPatterns = {
...
+    'mobile.mng-ads.com/'
+  ],
+  'LKQD': [
+    'ssp.lkqd.net/ad'
+  ],
+  'SARA': [
+    'ad.sara.media/hb'
+  ],
+  'RTBdemandADK': [
+    'cpm.metaadserving.com'
```
Comment (matthewlane) [inline bidderPatterns.js]: Need a trailing `/` for patterns that are hostnames only

File: bidderPatterns.js [hunk 2]
```diff
@@ -412,5 +413,118 @@ var bidderPatterns = {
...
+    't.visx.net/hb'
+  ],
+  'Unruly': [
+    'targeting.unrulymedia.com/prebid'
+  ],
+  'DigitalGarageAds': [
+    'ads-tr.bigmining.com/ad/p/bid'
+  ],
+  'Consumable': [
+    'adserver-us.adtech.advertising.com'
```
Comment (matthewlane) [inline bidderPatterns.js]: Add trailing `/`

File: bidderPatterns.js [hunk 3]
```diff
@@ -412,5 +413,118 @@ var bidderPatterns = {
...
+    'ads.gx1as.com/hb'
+  ],
+  'Clickonometrics': [
+    'delivery.clickonometrics.pl/ortb/prebid/bid'
+  ],
+  'Gamoshi Gambid': [
+    'rtb.gambid.io/r/*/bidr'
+  ],
+  'Papyrus': [
+    'prebid.papyrus.global'
```
Comment (matthewlane) [inline bidderPatterns.js]: Add trailing `/`

File: bidderPatterns.js [hunk 4]
```diff
@@ -412,5 +413,118 @@ var bidderPatterns = {
...
+  ],
+  'Andbeyond': [
+    'cpm.metaadserving.com'
```
Comment (matthewlane) [inline bidderPatterns.js]: Add trailing `/`

File: bidderPatterns.js [hunk 5]
```diff
@@ -412,5 +413,118 @@ var bidderPatterns = {
...
+    'ads.playground.xyz/host-config/prebid'
+  ],
+  'Fyber': [
+    'ad-tag.inner-active.mobi/simpleM2M/requestJsonAd'
+  ],
+  'Polymorph': [
+    'api.adsnative.com/v1/ad-template.json'
+  ],
+  'TripleLift': [
+    'tlx.3lift.com/header/auction'
```
Comment (matthewlane) [inline bidderPatterns.js]: This name and pattern already exist further up, can be removed

File: bidderPatterns.js [hunk 6]
```diff
@@ -412,5 +413,118 @@ var bidderPatterns = {
...
+    'ad.sara.media/hb'
+  ],
+  'RTBdemandADK': [
+    'cpm.metaadserving.com'
+  ],
+  'VIS.X': [
+    't.visx.net/hb'
+  ],
+  'Unruly': [
+    'targeting.unrulymedia.com/prebid'
```
Comment (matthewlane) [inline bidderPatterns.js]: This name and pattern already exist further up, can be removed

File: bidderPatterns.js [hunk 7]
```diff
@@ -412,5 +413,118 @@ var bidderPatterns = {
...
+    'c.deployads.com/openrtb2/auction'
+  ],
+  'AdUpTech': [
+    'rtb.d.adup-tech.com/prebid'
+  ],
+  'UOL': [
+    'prebid.adilligo.com/v1/prebid.json'
+  ],
+  'Zedo': [
+   'z2.zedo.com/asw/fmb.json'
```
Comment (matthewlane) [inline bidderPatterns.js]: Trailing space here can be removed

**Comment by jsnellbaker** [comment]
@matthewlane Thanks for the feedback; I just pushed in a commit that should include all the changes you requested.  Please let me know if it looks good now.

**Pull request merged**